test_data/semicolon_in_set.good.nix

```diff
@@ -10,12 +10,16 @@
   };
 
   c = [ x ]
-  ++ [ y ]
-  ++ [ z ]
-  ;
+    ++ [ y ]
+    ++ [ z ]
+    ;
```

Comment:
Hm, but this should not be indented, right?

https://github.com/nix-community/nixpkgs-fmt/blob/997a53a45c240887f60ea13b7b70a3f595bad9bf/test_data/set_multi_elem_value.good.nix#L3-L10

This seems slightly inconsistent.

Reply:
I think it should because it's not only a list literal that makes up the value.

That's the hard part because it's a bit inconsistent. But the alternative would be to indent all the values which would be worse.

Reply:
As an alternative, we mighth force a line break after `=`:

```nix
list = 
  [
    elem1
    elem2
    elem3
  ]
  ++ lib.optionals stdenv.isDarwin [ elem4 elem5 ]
  ++ lib.optionals stdenv.isLinux [ elem6 ]
;
```

Reply:
I would say that more people would use the put `=` and `[` on the same line in example for long `buildInputs`.